File: src/main/scala/com/aura/scala/db/JDBCHelper.scala
package com.aura.scala.db

import java.sql.{Connection, DriverManager}
import com.aura.java.config.Config

object JDBCHelper {
  
  def getConnection(): Connection = {
    Class.forName(Config.driver_class)
    return DriverManager.getConnection(Config.db_url, Config.username, Config.password)
	}
  
  def main(args: Array[String]): Unit = {
    JDBCHelper.getConnection()
  }
}

File: src/main/java/com/aura/java/db/JDBCHelper.java
package com.aura.java.db;

import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.SQLException;

import com.aura.java.config.Config;

public class JDBCHelper {
	
	public static Connection getConnection(){
		Connection conn = null;
		try {
			Class.forName(Config.driver_class);
			String url = Config.db_url;
			String username = Config.username;
			String password = Config.password;
			url += "&user=" + username + "&password=" + password;
			System.out.println(url);
			conn = DriverManager.getConnection(url);
		} catch (ClassNotFoundException e) {
			e.printStackTrace();
		} catch (SQLException e) {
			e.printStackTrace();
		}
		return conn;
	}
	
	public static void main(String[] args) {
		JDBCHelper.getConnection();
	}
}
